src/repr/src/bytes.rs

```diff
@@ -0,0 +1,243 @@
+// Copyright 2018 sqlparser-rs contributors. All rights reserved.
+// Copyright Materialize, Inc. and contributors. All rights reserved.
+//
+// This file is derived from the sqlparser-rs project, available at
+// https://github.com/andygrove/sqlparser-rs. It was incorporated
+// directly into Materialize on December 21, 2019.
```

Comment:
This doesn't look like the right copyright header.

Reply:
fixed 🤦‍♀️